runtime/flang/getarg.c

```diff
@@ -52,9 +53,9 @@ ENTRY(GETARGA, getarga)(__INT_T *n, DCHAR(arg) DCLEN64(arg))
 
 /* 32 bit CLEN version */
 void
-ENTRY(GETARG, getarg)(__INT_T *n, DCHAR(arg) DCLEN(arg))
+ENTF90(GETARG, getarg)(__INT_T *n, DCHAR(arg), __INT_T *int_kind DCLEN(arg))
 {
-  ENTRY(GETARGA, getarga)(n, CADR(arg), (__CLEN_T)CLEN(arg));
+  ENTF90(GETARGA, getarga)(n, CADR(arg), int_kind, (__CLEN_T)CLEN(arg));
 }
```

Comment:
Is this function needed anymore? Can it be removed, if there are no calls to it?

Reply:
As 1f4b8f14   committed by @mleair shows :

> Intrinsic routines that used to take a 32-bit CHARACTER length have been updated to the __CLEN_T type. In order to keep backwards compatibility we need to keep the old interface, and create a new routine with a new interface. The convention chosen is to append an "a" to the new routine.

Should we remove the old interface `GETARG`?

Reply:
My two cents: 1f4b8f1 added wrappers for a large number of runtime functions. The compiler should generate calls to the new versions of the functions when compiling new programs. I believe that the backward compatibility had to be maintained for the benefit of old programs that needed to work a new version of the runtime library without recompilation. The backward compatibility may be valuable for certain vendors/customers and we should probably discuss more carefully about how to deprecate the old symbols.

Reply:
OK. Fine with me.